### PR TITLE
Blog post on September Demo Day

### PR DESCRIPTION
**Pull request by matthewp**
## Changes

- This is a blog post on the September demo day.
- Includes links to the talks.
- Teases the upcoming release.

## Testing

N/A

## Docs

N/A

### CODE REVIEW

**Comment by tony-sull** [comment]
LGTM!  I like that the blog post is short and sweet, more of a teaser for the video than spoiling all the crazy new features demoed

File: www/src/pages/blog/demo-day-september-edition.astro [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  </head>
+
+  <body>
+    <BlogHeader />
+    <BlogPost title={title} author="matthew" heroImage="/assets/blog/introducing-astro.jpg" publishDate={publishDate}>
+      <Markdown>
+        **Astro Demo Day** was today! We had **4** amazing talks, including one big announcement on the future direction of Astro. If you missed it that's really unfortunate, but good news! You can see the full event [on YouTube](https://www.youtube.com/watch?v=-ExcBJrXOd8) or watch the talks in any order by visiting the links below. So you'll never miss another demo day please [subscribe to our channel](https://www.youtube.com/channel/UCeFjmvZEK-MoefuYXptnX6A), or better yet [join our Discord](https://astro.build/chat) and get involved. Today's demos were:
+
+        * [Developing the Astro VSCode extension](https://youtu.be/-ExcBJrXOd8?t=109) 🧑‍💻 from [Matthew Phillips](https://twitter.com/matthewcp)
+        * [Integrating a CMS - Astro+Forestry+Netlify](https://youtu.be/-ExcBJrXOd8?t=763) 🌳 from [Tony Sullivan](https://twitter.com/matthewcp)
```
Comment (tony-sull) [inline www/src/pages/blog/demo-day-september-edition.astro:32]: Update the twitter URL for https://twitter.com/navillus_dev

**Review by tony-sull — CHANGES_REQUESTED** [review]
Noticed a copy/paste error on the link for my talk, though I kind of appreciate the extra level of privacy it provides 😁

File: www/src/pages/blog/demo-day-september-edition.astro (hunk 1 above)
Comment (matthewp) [reply www/src/pages/blog/demo-day-september-edition.astro:32]: haha, i swear I wasn't trying to take credit for your talk :)

Comment (matthewp) [reply www/src/pages/blog/demo-day-september-edition.astro]: Updated

**Review by tony-sull — APPROVED** [review]

**Comment by natemoo-re** [comment]
Sorry for pushing on your branch @matthewp! Uploaded a social image and changed the post URL from `/demo-day-september-edition` to `/demo-day-2021-09` for clarity

**Comment by FredKSchott** [comment]
I like the desire to be mysterious in the final paragraph, but I think it comes off as unclear or maybe even a bit sneaky. "Wait, what's going on, what are they hiding from me???"

Given that the Vite news is officially out and we talked about having a blog post out later this week to announce it, I'd say either mention it briefly here or don't mention it at all. How about:

```
# astro@next

Astro's next release will be our biggest ever, featuring a new WASM-powered compiler plus a new Vite-powered runtime. We will publish a preview release to npm later today or tomorrow, for anyone to experiment with. Over the next few weeks, we will continue to polish the new code as we prepare for its official release as the next version of Astro!

If you’re not already, follow @astrodotbuild on Twitter for more news and announcements.
```

These demo days keep getting better; I can’t wait to see what we have in October!

File: www/src/pages/blog/demo-day-2021-09.astro [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+        * [Developing the Astro VSCode extension](https://youtu.be/-ExcBJrXOd8?t=109) 🧑‍💻 from [Matthew Phillips](https://twitter.com/matthewcp)
+        * [Integrating a CMS - Astro+Forestry+Netlify](https://youtu.be/-ExcBJrXOd8?t=763) 🌳 from [Tony Sullivan](https://twitter.com/navillus_dev)
+        * [The Astro SEO component](https://youtu.be/-ExcBJrXOd8?t=1384) 🗃 from [Jonas Schumacher](https://twitter.com/jonasmerlin1)
+        * [Astro's v2 Compiler and a Surprise](https://youtu.be/-ExcBJrXOd8?t=2070) 🪄 from [Drew Powers](https://twitter.com/drwpow) and [Nate Moore](https://twitter.com/n_moore)
+
+        As is always the case our community was very hype about the presentations.
```
Comment (FredKSchott) [inline www/src/pages/blog/demo-day-2021-09.astro:36]: "As is always the case" is unnecessary and makes the sentence harder to read. "about the presentations" could also be removed and the sentence wouldn't lose much.

Maybe: "As always, our Discord community brought the hype:"

File: www/src/pages/blog/demo-day-2021-09.astro [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+      }
+    </style>
+  </head>
+
+  <body>
+    <BlogHeader />
+    <BlogPost title={title} author="matthew" heroImage="/assets/blog/demo-day-2021-09/hero.png" publishDate={publishDate}>
+      <Markdown>
+        **Astro Demo Day** was today! We had **4** amazing talks, including one big announcement on the future direction of Astro. If you missed it that's really unfortunate, but good news! You can see the full event [on YouTube](https://www.youtube.com/watch?v=-ExcBJrXOd8) or watch the talks in any order by visiting the links below. So you'll never miss another demo day please [subscribe to our channel](https://www.youtube.com/channel/UCeFjmvZEK-MoefuYXptnX6A), or better yet [join our Discord](https://astro.build/chat) and get involved. Today's demos were:
```
Comment (FredKSchott) [inline www/src/pages/blog/demo-day-2021-09.astro:29]: "If you missed it that's really unfortunate, but good news!" is hard to read. Maybe just: "If you missed the livestream, fear not!"

Comment (FredKSchott) [reply www/src/pages/blog/demo-day-2021-09.astro:29]: "So you'll never miss another demo day please subscribe" is unnecessarily passive. 

Maybe: "Subscribe to the Astro YouTube channel to make sure you don't miss our next demo day."

File: www/src/pages/blog/demo-day-2021-09.astro [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+        * [The Astro SEO component](https://youtu.be/-ExcBJrXOd8?t=1384) 🗃 from [Jonas Schumacher](https://twitter.com/jonasmerlin1)
+        * [Astro's v2 Compiler and a Surprise](https://youtu.be/-ExcBJrXOd8?t=2070) 🪄 from [Drew Powers](https://twitter.com/drwpow) and [Nate Moore](https://twitter.com/n_moore)
+
+        As always, our Discord community brought the hype:
+
+        <img alt="Messages from the Demo Day chat room on Discord" src="/assets/blog/demo-day-2021-09/discord-chat.jpg" class="chat-messages" />
+
+        ## The next Astro release
+
+        Astro's next release will be our biggest ever, featuring a new WASM-powered compiler plus a new Vite-powered runtime. You can expect a preview release on [npm](https://www.npmjs.com/package/astro) later today or tomorrow. Over the next few weeks, we will continue to polish the new code as we prepare for its official release as the next version of Astro!
```
Comment (FredKSchott) [inline www/src/pages/blog/demo-day-2021-09.astro:42]: lol I know I wrote part of this originally, but re-reading I think "as the next version of Astro!" is duplicative, and not adding much info. safe to remove.

**Review by FredKSchott — APPROVED** [review]
LGTM!

**Pull request merged**